packages/paste-core/components/modal/__tests__/index.spec.tsx

```diff
@@ -96,6 +96,44 @@ const MockInitalFocusModal: React.FC = () => {
   );
 };
 
+const I18nMockModal: React.FC<{children?: React.ReactNode}> = ({children}) => {
+  const modalHeadingID = `modal-${useUID()}`;
+  return (
+    <CustomizationProvider baseTheme="default" theme={TestTheme}>
+      <Modal
+        data-testid="modal"
+        aria-busy="true"
+        id="a-new-id"
+        ariaLabelledby={modalHeadingID}
+        isOpen
+        onDismiss={handleCloseMock}
+        size="default"
+      >
+        <ModalHeader data-testid="modal-header" i18nDismissLabel="foo bar">
```

Comment:
thought (non-blocking): for the purpose of the test, I wonder if we can only render the `<ModalHeader>` to speed up the test.

Reply:
I tried only rendering the header, but it needs the `ModalContext.Provider` :/

Reply:
thought (non-blocking): Did you try `<Modal><ModalHeader ...></Modal>`? Essentially removing anything that can be removed from needing to render.

Reply:
Ok! I can do that.